statsmodels/tsa/tests/test_exponential_smoothing.py

```diff
@@ -618,3 +619,31 @@ def test_hessian(austourists_model):
     austourists_model.hessian(
         fit.params, approx_complex_step=False, approx_centered=True,
     )
+
+
+def test_convergence_simple():
+    # issue 6883
+    gen = np.random.default_rng(0)
```

Comment:
Need to use np.random.RandomState(0) here since we still support very old NumPy.